public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,78 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
+your own `Google Project` is required:
+
+1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
+    account.
+2.  Create `New Project` or select available one.
+3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
```

Comment:
it's not clear where this section is

Reply:
that Apis & services section should be picked. Let's add screenshots?

Reply:
can we actually go directly to `Credentials` in the `Apis & services`? or do we need to enable GDrive first. This instruction is confusing to be honest.

Reply:
It seems the order doesn't matter, but I've simplified the navigation instructions and added screenshots. Should look some easier to follow now.